package.json

```diff
@@ -48,27 +48,37 @@
   },
   "homepage": "https://github.com/SCE-Development/Core-v4#readme",
   "dependencies": {
+    "@grpc/proto-loader": "^0.5.3",
     "availity-reactstrap-validation": "^2.3.0",
     "axios": "^0.19.0",
     "bcrypt-nodejs": "0.0.3",
     "bluebird": "^3.7.0",
+    "body-parser": "^1.19.0",
     "bootstrap": "^4.1.3",
     "cors": "^2.8.5",
     "ejs": "^2.7.1",
     "es6-promise": "^4.2.8",
     "express": "^4.16.2",
     "filepond": "^4.9.3",
+    "filepond-plugin-file-encode": "^2.1.4",
+    "filepond-plugin-file-validate-type": "^1.2.4",
+    "google-protobuf": "^3.11.2",
+    "grpc": "^1.24.2",
     "isomorphic-fetch": "^2.2.1",
-    "mongodb": "^3.4.1",
-    "mongoose": "^5.8.3",
+    "mongodb": "^3.3.2",
+    "mongoose": "^5.7.3",
+    "multer": "^1.4.2",
     "node_hash": "^0.2.0",
     "passport": "^0.4.1",
     "passport-jwt": "^4.0.0",
+    "pdf-lib": "^1.3.1",
+    "pdfjs-dist": "^2.2.228",
     "react": "^16.10.1",
     "react-dom": "^16.10.1",
     "react-filepond": "^7.0.1",
     "react-iframe": "^1.8.0",
     "react-ionicons": "2.1.6",
+    "react-pdf": "^4.1.0",
```

Comment:
is this used anywhere or is it only in printingRPC? in that case we shouldn't need it?